arbitrum-docs/run-arbitrum-node/nitro/03-nitro-database-snapshots.md

```diff
@@ -0,0 +1,148 @@
+
+```shell
+wget -q -O - "$SNAPSHOT_URL".sha256
+```
+
+Once you know the expected snapshot checksum, run the command below to compute the checksum of the downloaded snapshot. Then, compare both add see if they are the same. If they are not the same, consider redownloading the snapshot. You must provide a valid snapshot to Nitro; otherwise, it won’t work properly.
+
+```shell
+sha256sum $PATH_TO_SNAPSHOT
+```
+
+Finally, you can provide path to the downloaded snapshot archive to Nitro using the `--init.url` flag, as described in the “Download the Snapshot from a URL” section.
+
+### Downloading snapshot parts
+
+If the snapshot is divided in parts, you should first download the manifest file in `<snapshot-url>.manifest.txt`. This manifest contains the names and checksums of each part. For instance, the snippet below shows how the manifest file should look like. You may use the commands described previously to download each part of the snapshot and verify their checksums.
+
+```shell
+a938e029605b81e03cd4b9a916c52d96d74c985ac264e2f298b90495c619af74  archive.tar.part0
+9e095ce82e70fa62bb6e7b4421e7f2c04b2cd9e21d2bc62cbbaaeb877408357b  archive.tar.part1
+e92172d6eaf770a76c7477e6768f742fc51555a5050de606bd0f837e59c7a61d  archive.tar.part2
+d1b6fb9aeeb23903cdbb2a7cca8e6909bff4ee8e51c8a5acac2a142b3e3a5437  archive.tar.part3
+f37e4552453202f2044e58b307bab7e466205bd280426abbc84f8646c6430cfa  archive.tar.part4
+972c5f513faca6ac4fadd22c70bea97707c6d38e9a646432bc311f0ca10497ed  archive.tar.part5
+```
+
+After downloading all the parts and verifying their checksums, you may use the command below to join them into a single archive.
+
+```shell
+cat archive.tar.part* > archive.tar
+```
+
+---
+
+## Extracting the snapshot manually
+
+It is also possible to extract the snapshot archive and place the files manually. First, you need to download the snapshot archive as described in “Manually Downloading the Snapshot”. Then, create the directory where Nitro will look for its database. By default, Nitro stores the database on `$HOME/.arbitrum/$CHAIN/nitro`. Move the archive to this directory and extract it. The commands below exemplify this process for the Arbitrum Sepolia chain.
+
+```shell
+export CHAIN=sepolia-rollup
+export ARCHIVE_PATH=/tmp/archive.tar.gz
+mkdir -p $HOME/.arbitrum/$CHAIN/nitro
+cd $HOME/.arbitrum/$CHAIN/nitro
+tar zxfv $ARCHIVE_PATH
+```
+
+You should see the following subdirectories in this directory after extracting the archive.
+
+```shell
+arbitrumdata
+l2chaindata
+nodes
+```
+
+---
+
+## Creating a snapshot
+
+To generate the a snapshot for the Nitro database, first you need to stop the process gracefully. You must not generate the snapshot while Nitro is running because the database might be in a intermediary state. Nitro should print logs like the ones described below when stopping.
```

Comment:
```suggestion
To generate a snapshot for the Nitro database, you first need to stop the process gracefully. You must not generate the snapshot while Nitro runs because the database might be in an intermediary state. Nitro should print logs like the ones described below when stopping.
```